### ISSUE TITLE
通过ERNIE finetuning后利用--inference_model_dir参数保存的模型无法用Paddle Inference去加载预测

### ISSUE THREAD

**Issue by xiaoranchenwai** [PaddlePaddle/ERNIE#690]
训练过程中添加了args.inference_model_dir参数如下：
    if args.inference_model_dir is not None:
        class InferenceModel(ErnieModelForSequenceClassification):
            def forward(self, ids, sids):
                _, logits = super(InferenceModel, self).forward(ids, sids)
                return logits
        model.__class__ = InferenceModel
        log.debug('saving inference model')
        src_placeholder = P.zeros([2, 2], dtype='int64')
        sent_placehodler = P.zeros([2, 2], dtype='int64')
        _, static = P.jit.TracedLayer.trace(
            model, inputs=[src_placeholder, sent_placehodler])
        static.save_inference_model(str(args.inference_model_dir))
        log.debug('done')
最后保存的文件格式为：inference_model.pdiparams和inference_model.pdmodel两个文件，无法用inference中的代码去加载预测。

**Comment by xuyifan-0731** [comment]
你好，请问这个问题是怎么解决的啊？我也遇到这个问题了
